dcs/triggers.py

```diff
@@ -197,6 +202,7 @@ class TriggerRule:
 
     def __init__(self, event: Event, comment: str):
         self.comment: str = comment
+        self.color: str = "0x00000000"
```

Comment:
and then also use the `TriggerColor.White` here

Reply:
Hello,
This has been done in 18eb279!
Thanks.

Reply:
Seems I looked at the wrong diff, sorry